src/shared/components/TrackIcon/style.scss

```diff
@@ -55,19 +55,19 @@ $track-radius-4: $corner-radius * 2;
 
     &.design,
     &.generic {
-      color: $tc-light-blue;
+      color: $tc-light-blue-110;
```

Comment:
@r0hit-gupta Same here it doesn't meet the ratio

using tc-dark-blue-110 for text meets this.